fiobank.py

```diff
@@ -1,10 +1,10 @@
 # -*- coding: utf-8 -*-
 
 import re
-from datetime import datetime, date
-
 import requests
 
+from datetime import datetime, date
```

Comment:
It might only my thing but I usually do imports in following way:

import ...

from libs import ...  - libs like django etc.
from core import ... - project modules

Usually its nicer to look at because from ... import ... are much longer than simple imports... But overall it does not matter, its just my kind of fetish :D